### PR TITLE
Added examples in readme for azurestackhci

### PR DESCRIPTION
**Pull request by NikitaSingh-ms**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by jsquire** [comment]
@NikitaSingh-ms:

Our automation is identifying you as a community contributor.  Please review the Azure SDK onboarding documentation and ensure that your GitHub organization membership and permissions are correct.
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

File: sdk/azurestackhci/Azure.ResourceManager.Hci/README.md [hunk 1]
```diff
@@ -43,6 +43,299 @@ Documentation is available to help you learn how to use this package:
 - [Authentication](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/README.md).
 
 ## Examples
+#### Prerequisites
```
Comment (ArthurMa1978) [inline sdk/azurestackhci/Azure.ResourceManager.Hci/README.md:46]: Please follow https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/README.md?plain=1#L44 to add samples by using code Snippet which can make sure the code is compilable.

Comment (NikitaSingh-ms) [reply sdk/azurestackhci/Azure.ResourceManager.Hci/README.md:46]: Resolved, please review

Comment (ArthurMa1978) [reply sdk/azurestackhci/Azure.ResourceManager.Hci/README.md:46]: You need add a **cs file** with the sample code, like this https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/samples/DocSamples/Readme.cs, then add the `C# Snippet` link with the corresponding code snippet, then run `eng\scripts\Update-Snippets.ps1 azurestackhci` to update the code from the **cs file**.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**